src/air/responses.py

```diff
@@ -23,3 +23,17 @@ def render(self, content: Any) -> bytes:
         if isinstance(content, dict):
             content = dict_to_ft_component(content)
         return content.render().encode("utf-8")
+
+
+class AirResponse(Response):
+    """Response class to handle air.tags.Tags or HTML (from Jinja2)."""
+
+    media_type = "text/html; charset=utf-8"
+
+    def render(self, content: Any) -> bytes:
+        """Render Tag elements to bytes of HTML."""
+        if isinstance(content, str):
+            return content.encode("utf-8")
+        if isinstance(content, dict):
+            content = dict_to_ft_component(content)
+        return content.render().encode("utf-8")
```

Comment:
Should we explicitly handle edge case if `content` does not have a `render` method?  Might be overkill.

Reply:
I'll open a new ticket so we can figure that out.